src/components/InternalLessonCard/InternalLessonCard.tsx

```diff
@@ -0,0 +1,106 @@
+import * as React from 'react';
+import { styled, ThemeProvider } from '@mui/material/styles';
+import theme from '../../styles/theme/theme';
+import { CircleIconButton } from '../CircleIconButton/CircleIconButton';
+import { Label } from '../Label/Label';
+import { Grid } from '@mui/material';
+import { Typography } from '@mui/material';
+import ArrowCircleRightIcon from '@mui/icons-material/ArrowCircleRight';
+
+interface InternalLessonCard {
```

Comment:
I believe we said interface names need to be looked at again (i.e. the one has the same name as the react component).